lib/utils/__tests__/isStandardSyntaxAtRule.test.js

```diff
@@ -1,136 +1,112 @@
 
 	it('nested at-rules with windows newline after name', () => {
-		atRules('@media\r\n(min-width: 100px) {};', (atRule) => {
-			expect(isStandardSyntaxAtRule(atRule)).toBeTruthy();
-		});
+		expect(isStandardSyntaxAtRule(atRules('@media\r\n(min-width: 100px) {};')[0])).toBeTruthy();
 	});
 
 	it('nested at-rules without space after name', () => {
-		atRules('@media(min-width: 100px) {};', (atRule) => {
-			expect(isStandardSyntaxAtRule(atRule)).toBeTruthy();
-		});
+		expect(isStandardSyntaxAtRule(atRules('@media(min-width: 100px) {};')[0])).toBeTruthy();
 	});
 
-	it('ignore `@content` inside mixins newline', () => {
-		const sass = '@mixin mixin()\n  @content';
+	// eslint-disable-next-line jest/no-disabled-tests -- TODO: `postcss-sass` parser does not support `@mixin`.
+	it.skip('ignore `@content` inside mixins newline', () => {
+		const rules = sassAtRules('@mixin mixin()\n  @content');
 
-		sassAtRules(sass, (atRule) => {
-			if (atRule.name === 'mixin') {
-				return;
-			}
-
-			expect(isStandardSyntaxAtRule(atRule)).toBeFalsy();
-		});
+		expect(rules).toHaveLength(2);
+		expect(rules.map((rule) => rule.name)).toEqual(['mixin', 'content']);
+		expect(isStandardSyntaxAtRule(rules[0])).toBeTruthy();
+		expect(isStandardSyntaxAtRule(rules[1])).toBeFalsy();
 	});
 
 	it('ignore `@content` inside mixins space', () => {
-		const scss = '@mixin mixin() { @content; };';
-
-		scssAtRules(scss, (atRule) => {
-			if (atRule.name === 'mixin') {
-				return;
-			}
+		const rules = scssAtRules('@mixin mixin() { @content; };');
 
-			expect(isStandardSyntaxAtRule(atRule)).toBeFalsy();
-		});
+		expect(rules).toHaveLength(2);
+		expect(rules.map((rule) => rule.name)).toEqual(['mixin', 'content']);
+		expect(isStandardSyntaxAtRule(rules[0])).toBeTruthy();
+		expect(isStandardSyntaxAtRule(rules[1])).toBeFalsy();
 	});
 
 	it('ignore passing rulesets to mixins', () => {
-		const less = '@detached-ruleset: { background: red; }; .top { @detached-ruleset(); }';
+		const rules = lessAtRules(
+			'@detached-ruleset: { background: red; }; .top { @detached-ruleset(); }',
+		);
 
-		lessAtRules(less, (atRule) => {
-			expect(isStandardSyntaxAtRule(atRule)).toBeFalsy();
-		});
+		expect(rules).toHaveLength(2);
+		expect(isStandardSyntaxAtRule(rules[0])).toBeFalsy();
+		expect(isStandardSyntaxAtRule(rules[1])).toBeFalsy();
 	});
 
 	it('ignore calling of mixins', () => {
-		const less = 'a { .mixin(); }';
+		const rules = lessAtRules('a { .mixin(); }');
 
-		lessAtRules(less, (atRule) => {
-			expect(isStandardSyntaxAtRule(atRule)).toBeFalsy();
-		});
+		expect(rules).toHaveLength(1);
+		expect(isStandardSyntaxAtRule(rules[0])).toBeFalsy();
 	});
 
 	it('ignore variables', () => {
-		const less = `
-        @my-variable: 10px;
-        .top { margin-top: @my-variable; }
-    `;
-
-		lessAtRules(less, (atRule) => {
-			expect(isStandardSyntaxAtRule(atRule)).toBeFalsy();
-		});
+		const rules = lessAtRules('@my-variable: 10px; .top { margin-top: @my-variable; }');
```

Comment:
[note] I think the one-line is easier to read here such as `'@detached-ruleset: { background: red; }; .top { @detached-ruleset(); }'`.